src/pc/1.13/BitArray.test.js

```diff
@@ -0,0 +1,125 @@
+/* globals describe it BigInt */
```

Comment:
Why are you putting the test file in the same folder as source ?

Reply:
I like having my tests closer to the source. I feel like it makes me not forget to add tests and it's clearer to see what things are missing tests. More of a personal opinion really but a dedicated test folder tends to get messy after a while. This is only concerning unit tests though.